backend/core/src/main/java/org/sonarsource/sonarlint/core/analysis/AnalysisService.java

```diff
@@ -416,7 +420,9 @@ private List<ActiveRuleDto> buildStandaloneActiveRules() {
 
     var filteredActiveRules = new ArrayList<SonarLintRuleDefinition>();
 
-    var allRulesDefinitions = rulesRepository.getEmbeddedRules();
+    var allRulesDefinitions = rulesRepository.getEmbeddedRules().stream()
+      .filter(rule -> !hotspotsOnly || !rule.getType().equals(RuleType.SECURITY_HOTSPOT))
+      .collect(toList());
```

Comment:
Hotspots are not supported in standalone mode, are they? I find it strange that the 2nd part of the condition was not already there. The first part can be removed